Re-work sepolicy for graphic driver

renchenglei opened the pull request:
Remove some un-used library from sepolicy, and change gralloc
sepolicy from "intel" to "celadon".

Tracked-On: OAM-92961
Signed-off-by: Ren Chenglei <chenglei.ren@intel.com>

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/2439

renchenglei committed:
Re-work sepolicy for graphic driver

Remove some un-used library from sepolicy, and change gralloc
sepolicy from "intel" to "celadon".

Tracked-On: OAM-92961
Signed-off-by: Ren Chenglei <chenglei.ren@intel.com>

renchenglei commented:
@kwang13, could you help take a review of this changes? Thanks a lot!

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckBug Bad comments/Bugs

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/2456

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckBug Bad comments/Bugs

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/2457

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/2458

Done by "Ren, Chenglei" <chenglei.ren@intel.com>

sysopenci reviewed (commented):
PR integrated into 'premerge' builder



More details, logs, and binaries: /absp/builders/celadon-premerge/builds/1230

Done by "Ren, Chenglei" <chenglei.ren@intel.com>

sysopenci reviewed (commented):
cactus finished building and testing your changes 

It was requested for merge by cactusi@intel.com 

so we merge it. Thanks for your contribution.

more details, logs, and binaries: /absp/builders/celadon-mergerequest/builds/1967
